src/Button.js

```diff
@@ -7,27 +7,23 @@ const size = props => {
   switch (props.size) {
     case 'small':
       return {
-        height: '32px',
         fontSize: `${props.theme.fontSizes[0]}px`,
-        padding: '0 12px'
+        padding: '7px 12px'
       }
     case 'medium':
       return {
-        height: '40px',
         fontSize: `${props.theme.fontSizes[1]}px`,
-        padding: '0 18px'
+        padding: '9.5px 18px'
       }
     case 'large':
       return {
-        height: '48px',
         fontSize: `${props.theme.fontSizes[2]}px`,
-        padding: '0 22px'
+        padding: '12px 22px'
       }
     default:
       return {
-        height: '40px',
         fontSize: `${props.theme.fontSizes[1]}px`,
-        padding: '0 18px'
+        padding: '9.5px 18px'
```

Comment:
In order to get our button heights to be 32px (small), 40px (medium), and 48px (large) we may have odd numbers for top/bottom padding. The reason line-height and padding is a smart solution though is incase the button wraps to 2 lines we keep a consistent look and feel.

Reply:
Just a curious none-blocking question. Read @odmln' use case & explaining to @msafari about the 2-lines wrapping, consistent look and feel with the technique of "_.5" in CSS `px` unit.

My question is that - while "_.5" is working, is there another alternative to use maths operation such as `%` operator against the defined `small`, `medium`, `large` namespaces with the `px`? (i.e. let's take "xyz" as an arbitrary number of px, and the math uses the mod operator `%` like `xyz % 32` to decide the decimal offset that perhaps could fit into the consistent look and feel UI requirement here. I guess my sentiment is that instead of `.5` in px, maybe the code can apply some simple maths to drive the consistent look and feel mathematical & systematically. (Happy to open an issue to discuss this idea with you more, if you're interested @odmln. And keep this PR's current scope as it is, of course.